Manual.md

```diff
@@ -1302,6 +1326,360 @@ cache during the install/removal of the package
+
+This is not a complete overview of the package. It is recommended to read the variables
+referenced and the triggers themselves.
+
+<a id="triggers_appstream_cache"></a>
+#### appstream-cache
+
+The appstream-cache trigger is responsible for rebuilding the appstream metadata cache.
+
+During installation it executes `appstreamcli refresh-cache --verbose --force --datapath
+$APPSTREAM_PATHS --cachepath var/cache/app-info/gv`. By default APPSTREAM_PATHS are all the
+paths that appstreamcli will look into for metadata files.
+
+The directories searched by appstreamcli are:
+
+- `usr/share/appdata`
+- `usr/share/app-info`
+- `var/lib/app-info`
+- `var/cache/app-info`
+
+During removal of the `AppStream` package it will remove the `var/cache/app-info/gv`
+directory.
+
+It is automatically added to packages that have XML files under one of the directories
+searched by appstreamcli.
+
+<a id="triggers_binfmts"></a>
+#### binfmts
+
+The binfmts trigger is responsible for registration and removal of arbitrary
+executable binary formats, know as binfmts.
+
+During installation/removal it uses `update-binfmts` from the `binfmt-support` package
+to register/remove entries from the arbitrary executable binary formats database.
+
+To include the trigger use the `binfmts` variable, as the trigger won't do anything unless
+it is defined.
+
+<a id="triggers_dkms"></a>
+#### dkms
+
+The dkms trigger is responsible for compiling and removing dynamic kernel modules of a
+package.
+
+During installation the trigger compiles and install the dynamic module for all `linux`
+packages that have their corresponding headers installed. During removal the corresponding
+module will be removed
+
+To include the trigger use the `dkms_modules` variable, as the trigger won't do anything
+unless it is defined.
+
+<a id="triggers_gconf_schemas"></a>
+#### gconf-schemas
+
+The gconf-schemas trigger is responsible for registering and removing .schemas and
+.entries files into the schemas database directory
+
+During installation it uses `gconftool-2` to install .schemas and .entries files into
+`usr/share/gconf/schemas`. During removal it uses `gconftool-2` to remove the entries
+and schemas from the database from the package being removed.
+
+To include it add `gconf-schemas` to `triggers` and add the appropriate .schemas in
+the `gconf_schemas` variable and .entries in `gconf_entries`.
+
+It is automatically added to packages that have `/usr/share/gconf/schemas` present
+as a directory. All files with the schemas extension under that directory are passed
+to the trigger.
+
+<a id="triggers_gdk_pixbuf_loaders"></a>
+#### gdk-pixbuf-loaders
+
+The gdk-pixbuf-loaders trigger is responsible for maintaining the GDK Pixbuf loaders cache.
+
+During installation it runs `gdk-pixbuf-query-loaders --update-cache` and also deletes
+the obsolete `etc/gtk-2.0/gdk-pixbuf.loaders` file if present. During removal of the
+gdk-pixbuf package it removes the cache file, if present, present at
```

Comment:
This isn't an essay, so I guess it doesn't matter, but two times `present` right next to each other sounds a little "meh". Also not quite sure if those commas are correct.